.github/workflows/test.yml

```diff
@@ -17,5 +17,14 @@ jobs:
         with:
           go-version: 1.19
           check-latest: true
+      - name: Setup node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
```

Comment:
Please add a version in a comment in the same line. See the above for example

Thanks